lib/src/chip/Chip.stories.tsx

```diff
@@ -50,33 +44,56 @@ const opinionatedTheme = {
 export const Chromatic = () => (
   <>
     <ExampleContainer>
-      <Title title="Basic chip" theme="light" level={4} />
-      <DxcChip label="Default Chip" />
+      <Title title="With prefix (SVG)" theme="light" level={4} />
+      <DxcChip label="Prefix" prefixIcon={iconSVG} />
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="With action prefix (SVG)" theme="light" level={4} />
+      <DxcChip label="Action prefix" prefixIcon={iconSVG} onClickPrefix={() => {}} />
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Disabled action prefix (SVG)" theme="light" level={4} />
+      <DxcChip label="Disabled action prefix" prefixIcon={iconSVG} onClickPrefix={() => {}} disabled />
     </ExampleContainer>
     <ExampleContainer>
-      <Title title="Chip with prefix SVG (small icon)" theme="light" level={4} />
-      <DxcChip label="Chip with prefix" prefixIcon={smallIconSVG} />
+      <Title title="With suffix (Material icon)" theme="light" level={4} />
+      <DxcChip label="Suffix" suffixIcon="filled_check_circle" />
     </ExampleContainer>
     <ExampleContainer>
-      <Title title="Chip with suffix SVG (large icon)" theme="light" level={4} />
-      <DxcChip label="Chip with suffix" suffixIcon={iconSVG} />
+      <Title title="With action suffix (Material icon)" theme="light" level={4} />
+      <DxcChip label="Action suffix" suffixIcon="filled_check_circle" onClickSuffix={() => {}} />
     </ExampleContainer>
     <ExampleContainer>
-      <Title title="Chip with prefix (SVG) and suffix (URL)" theme="light" level={4} />
-      <DxcChip label="Chip with prefix and suffix" prefixIcon={iconSVG} suffixIcon="filled_check_circle" />
+      <Title title="Action prefix and suffix" theme="light" level={4} />
+      <DxcChip
+        label="Action prefix and suffix"
+        prefixIcon="filled_check_circle"
+        onClickPrefix={() => {}}
+        suffixIcon={iconSVG}
+      />
```

Comment:
Could you add an example with both prefix and suffix with the suffix being the one with the action?

Reply:
Done!